### PR TITLE
feat: add elb logtank

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=
'./huaweicloud/services/acceptance/elb' TESTARGS='-run=TestAccElbLogTank_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/elb -v -run=TestAccElbLogTank_basic -timeout 360m -parallel 4
=== RUN   TestAccElbLogTank_basic 
=== PAUSE TestAccElbLogTank_basic 
=== CONT  TestAccElbLogTank_basic 
--- PASS: TestAccElbLogTank_basic (99.60s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/elb       99.654s 
```

### CODE REVIEW

File: docs/resources/elb_logtank.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the logtank resource.
+  If omitted, the provider-level region will be used. Changing this creates a new logtank.
+
+* `loadbalancer_id` - (Required, String) Specifies the ID of a loadbalancer. It can not be changed.
```
Comment (hmbig2) [inline docs/resources/elb_logtank.md]:
```suggestion
* `loadbalancer_id` - (Required, String, ForceNew) Specifies the ID of a loadbalancer. Changing this creates a new logtank.
```

File: huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func checkResourceDestroy(s *terraform.State) error {
```
Comment (hmbig2) [inline huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go]: delete  this function

File: huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&logTanks,
+		getELBLogTankResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      checkResourceDestroy,
```
Comment (hmbig2) [inline huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go]:
```suggestion
		CheckDestroy:      rc.checkResourceDestroy,
```

File: huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+func checkGroupIdAndTopicId(config *config.Config, d *schema.ResourceData) diag.Diagnostics {
+	logStreamClient, err := config.LtsV2Client(config.GetRegion(d))
+	if err != nil {
+		return diag.Errorf("error creating LTS client: %s", err)
+	}
+	logGroupId := d.Get("log_group_id").(string)
+	logTopicId := d.Get("log_topic_id").(string)
+	streams, err := logstreams.List(logStreamClient, logGroupId).Extract()
+	if err != nil {
+		if _, ok := err.(golangsdk.ErrDefault400); ok {
```
Comment (hmbig2) [inline huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go:162]: no matter what error, just return the error directly?

File: docs/resources/elb_logtank.md (hunk 1 above)
Comment (houpeng80) [reply docs/resources/elb_logtank.md]: done

File: huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go (hunk 3 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go]: done

File: huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_logtank_test.go]: done

File: huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go (hunk 4 above)
Comment (houpeng80) [reply huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go:162]: the input param log_group_id may be wrong, so it is better to describe separately

File: huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+			return diag.Errorf("the log group id %s is error: %s", logGroupId, err)
+		}
+		return diag.Errorf("error getting LTS log stream by log group id %s: %s", logGroupId, err)
+	}
+	containLogTopicId := false
+	for _, stream := range streams.LogStreams {
+		if stream.ID == logTopicId {
+			containLogTopicId = true
```
Comment (ShiChangkuo) [inline huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go:170]: `break` is missing

File: docs/resources/elb_logtank.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+Manage an ELB logtank resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "loadbalancer_id" {}
+variable "group_id" {}
+variable "topic_id" {}
+resource "huaweicloud_elb_logtank" "test" {
```
Comment (ShiChangkuo) [inline docs/resources/elb_logtank.md]:
```suggestion
variable "topic_id" {}

resource "huaweicloud_elb_logtank" "test" {
```

please add a new line

File: huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go (hunk 5 above)
Comment (houpeng80) [reply huaweicloud/services/elb/resource_huaweicloud_elb_logtank.go:170]: done

File: docs/resources/elb_logtank.md (hunk 6 above)
Comment (houpeng80) [reply docs/resources/elb_logtank.md]: done

**Pull request merged**